components/dashboard/src/experiments/configcat.ts

```diff
@@ -35,6 +36,7 @@ export function newNonProductionConfigCatClient(): Client {
     // clientKey is an identifier of our ConfigCat application. It is not a secret.
     const clientKey = "WBLaCPtkjkqKHlHedziE9g/LEAOCNkbuUKiqUZAcVg7dw";
     const client = configcat.createClient(clientKey, {
+        maxInitWaitTimeSeconds: 0,
```

Comment:
What's the purpose of the default wait time of 5 seconds? Any downsides to setting it to 0?

Reply:
I have no idea -- there seems to be no documentation for this feature apart from the one-line description shown in my PR description (see also ConfigCat's [source code](https://github.com/configcat/common-js/blob/14cab5585a4fc8bbe79c7c038ac0882b18b05ec9/src/ConfigCatClientOptions.ts#L127-L128))

Digging a bit into ConfigCat's history, it looks like this feature was already part of this [initial commit](https://github.com/configcat/common-js/commit/7d3cdc7d0796afbb434e0fbda7805581370298ff), which unfortunately doesn't provide much context as to why this feature is useful. 🙈